_posts/en/newsletters/2019-12-11-newsletter.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Bitcoin Optech Newsletter #76'
+permalink: /en/newsletters/2019/12/11/
+name: 2019-12-11-newsletter
+slug: 2019-12-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a new maintenance release of LND,
+summarizes a discussion about watchtowers for eltoo payment channels,
+and describes several notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+<!-- FIXME: I think this will be released before Wednesday; if not, we
+can make this a help-test-the-RC item -->
+
+- **Upgrade to LND 0.8.2-beta:** this [release][lnd 0.8.2-beta] contains
+  several bug fixes and minor UX improvements, most notably for the
+  recovery of static channel backups.  (One of these improvements is
+  [described][lnd-3698] later in this newsletter.)
+
+## News
+
+- **Watchtowers for eltoo payment channels:** [eltoo][topic eltoo] is a
+  proposed alternative payment channel layer for LN that doesn't require
+  participants be able to generate penalty transactions.
```

Comment:
I think "be able" is correct, since the penalty transaction only needs to be created on demand.  For example, in a non-watchtower flow Bob only generates the penalty transaction if he sees Mallory publish an old state.  This is possible because the penalty transaction isn't pre-signed: it depends on Bob using the commitment revocation private key he received from Mallory when she agreed to revoke an old state.

Reply:
_require participants be able_ is also a nice use of the rare (in English) subjunctive mood, so we should definitely keep it 🙂